script/breakupScript.py

```diff
@@ -0,0 +1,65 @@
+import sys
+
+import numpy
+import numpy as np
+import scipy as sp
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import pandas as pd
+
+
+def histogram_lc(data, title, cumulative):
+    lc = data["Characteristic Length [m]"]
+    weights = np.ones_like(lc) / float(len(lc))
+    plt.hist(lc, density=False, bins=1000, color='b', weights=weights, cumulative=cumulative)
+    plt.xscale("log")
+    plt.xlabel('$L_c [m]$')
+    plt.ylabel('Relative Probability')
+    if cumulative:
+        plt.title(title + ' $L_c$ Cumulative Distribution')
+    else:
+        plt.title(title + ' $L_c$ Distribution')
+    plt.show()
+
+
+def histogram_am(data, title, cumulative):
+    am = data["A/M [m^2/kg]"]
+    weights = np.ones_like(am) / float(len(am))
+    plt.hist(am, density=False, bins=5000, color='b', weights=weights, cumulative=cumulative)
+    plt.xscale("log")
+    plt.xlim(10e-4, 10)
+    plt.xlabel('$\log(A/M) [m^2/kg]$')
+    plt.ylabel('Relative Probability')
+    if cumulative:
+        plt.title(title + ' $A/M$ Cumulative Distribution')
+    else:
+        plt.title(title + ' $A/M$ Distribution')
+    plt.show()
+
+
+def scatter_lc_am(data, title):
```

Comment:
if you want to be super correct you can add docstrings see e.g. [here](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html).